### PR TITLE
various fixes motivated by a failure of the raw syscall test

### PR DESCRIPTION
**Pull request by kkourt**
see commits

### CODE REVIEW

File: pkg/selectors/selectors.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func WriteSelectorUint32(k *KernelSelectorState, v uint32) {
 
 func WriteSelectorInt64(k *KernelSelectorState, v int64) {
 	binary.LittleEndian.PutUint64(k.e[k.off:], uint64(v))
-	k.off += 4
+	k.off += 8
```
Comment (tixxdz) [inline pkg/selectors/selectors.go:38]: Could this be covered by simple tests?

**Review by tixxdz — APPROVED** [review]
Thank you @kkourt

File: pkg/selectors/selectors.go (hunk 1 above)
Comment (jrfastab) [reply pkg/selectors/selectors.go:38]: ouch :( guess we don't use int64 much.

Comment (kkourt) [reply pkg/selectors/selectors.go:38]: > Could this be covered by simple tests?

The test is in the PR :)

**Pull request merged**